src/signals/incident-management/containers/Dashboard/components/AreaChart.tsx

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import vegaEmbed from 'vega-embed'
+import type { EmbedOptions } from 'vega-embed'
+
+import { getAreaChart } from '../charts'
+import { constants } from '../charts'
+import { ModuleTitle } from '../components/ModuleTitle'
+import { ComparisonRate } from './ComparisonRate'
+import { AreaChartWrapper as Wrapper } from './styled'
+import { getMaxDomain } from './utils'
+
+/**
+ * Mock data. Should be retrieved form backend when ready.
+ */
+const mockToday = { year: 2012, month: 1, date: 7, hours: 23, minutes: 0 }
+const mockPercentage = 12
+const mockValues = [
+  { date: '01 Jan 2012 23:00:00', amount: 48 },
```

Comment:
Ik had als test de data veranderd van 1 januari naar 8 januari en de mockToday op date 8 gezet. Ik krijg dit: 
![image](https://user-images.githubusercontent.com/25867841/215754706-b26ca331-e83d-45aa-adcc-49f923a91e64.png)
Ik verwachte dat de dagen eentje zouden verspringen.

Reply:
Not sure what you used as input, but when I create mock data based on today with a large amount on de second day it comes out great!
It did sort on Sunday to Saturday though, so I set sort to null. Good catch!

<img width="1549" alt="Screenshot 2023-01-31 at 13 38 21" src="https://user-images.githubusercontent.com/46756331/215762412-7db1b634-ca6a-42c1-a4ef-9a1045a6c18f.png">

Reply:
Ha Vincent,

Ik krijg niet meer de rare vlek zonder kleur so that's good. Wel krijg ik nu dat het aantal wat aan het einde van de week staat verspringt naar het beginpunt. Zie het plaatje: 
![image](https://user-images.githubusercontent.com/25867841/216005528-0c9e55b7-aab1-4007-9f17-d416cede0fd0.png)


Ik heb de volgende data veranderd: 
![image](https://user-images.githubusercontent.com/25867841/216005759-28df212f-e1ee-4846-9e24-bd813544512f.png)

Reply:
That's right, you need to add the data the correct way.

Reply:
Top. Nog één vraagje over dat percentageding (zie hieronder en dan ben je daarna wat mij betreft good to go).